server/jetstream_cluster.go

```diff
@@ -2078,9 +2087,19 @@ func (js *jetStream) createRaftGroup(accName string, rg *raftGroup, storage Stor
 	storeDir := filepath.Join(js.config.StoreDir, sysAcc.Name, defaultStoreDirName, rg.Name)
 	var store StreamStore
 	if storage == FileStorage {
+		// If the server is set to sync always, do the same for the Raft log.
+		js.srv.optsMu.RLock()
+		syncAlways := js.srv.opts.SyncAlways
+		syncInterval := js.srv.opts.SyncInterval
```

Comment:
Does this match 5 mins? Noticed this was slightly more delayed then default 2m.

Reply:
They will now match the default 2 minutes. There's nothing in the git history that explains why it was 5 minutes.

Reply:
We probably would like to not have them firing at the same time.

Reply:
I've pushed another commit with an idea, curious on your thoughts: what if the stores used a random interval between 0 and `fs.fcfg.SyncInterval` for the first timer fire, and then tick along on `fs.fcfg.SyncInterval` after that? That way stores syncing should be spread out more.